SPEC.md

```diff
@@ -762,8 +764,8 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
         * **count** (integer, optional, defaults to 1) specifies a range of ports, starting with "port" and ending with "port" + "count" - 1.
         * **socketActivated** (boolean, optional, defaults to "false" if unsupplied) if set to true, the application expects to be [socket activated](http://www.freedesktop.org/software/systemd/man/sd_listen_fds.html) on these ports. The ACE must pass file descriptors using the [socket activation protocol](http://www.freedesktop.org/software/systemd/man/sd_listen_fds.html) that are listening on these ports when starting this app. If multiple apps in the same pod are using socket activation then the ACE must match the sockets to the correct apps using getsockopt() and getsockname().
 * **dependencies** (list of objects, optional) dependent application images that need to be placed down into the rootfs before the files from this image (if any). The ordering is significant. See [Dependency Matching](#dependency-matching) for how dependencies are retrieved.
-    * **app** (string, required) name of the dependent App Container Image.
-    * **imageID** (string of type [Image ID](#image-id-type), optional) content hash of the dependency. If provided, the retrieved dependency must match the hash. This can be used to produce deterministic, repeatable builds of an App Image that has dependencies.
+    * **name** (string of type [AC Name](#ac-name-type), required) name of the dependent App Container Image.
```

Comment:
@cdaylward my initial argument in favour of `name` would be that it _is_ a key reference. But on thinking about it further it seems like:
a) this field should be optional; one might desire a deterministic set of dependencies (i.e. only referencing by ID) and the name field is not then strictly necessary (provided of course the implementation supports lookups based on ID alone)
b) a qualified `imageName` / `imageID` might be preferable since these are not actually image objects (rather, dependency objects)

@yifan-gu thoughts?